apps/rush-lib/src/logic/GitPolicy.ts

```diff
@@ -33,19 +28,32 @@ If you didn't configure your e-mail yet, try something like this:`);
       console.log(colors.cyan(
 `
     git config --local user.name "Mr. Example"
-    git config --local user.email "${rushConfiguration.gitSampleEmail}"
+    git config --local user.email "${rushConfiguration.gitSampleEmail || "example@contoso.com"}"
 `));
 
       console.log(colors.red('Aborting, so you can go fix your settings.  (Or use --bypass-policy to skip.)'));
 
-      return false;
+      return undefined;
     }
 
     // sanity check; a valid e-mail should not contain any whitespace
     if (!userEmail.match(/^\S+$/g)) {
       throw new Error('The gitPolicy check failed because "git config" returned unexpected output:'
```

Comment:
>throw [](start = 6, length = 5)

Should this just print the error and return `undefined`?

Reply:
yeah probably

---
In reply to: [201196353](https://github.com/Microsoft/web-build-tools/pull/740#discussion_r201196353) [](ancestors = 201196353)